requirements_dev.txt

```diff
@@ -1,10 +1,11 @@
 -r requirements.txt
 wheel
+astroid>=3.2.0
 pylint>=2.16.0
```

Comment:
Thanks for the shout out. I'm contributing less often to pylint these days, but what I remember from when I did more of it was advising folks not to manage their own astroid dependency. There's a very narrow band of compatibility between pylint and astroid. They're independent projects in name only, a speculative decision made many eons ago.

Long story short, I'd just pin pylint >= 3.2.0 (which requires astroid 3.2) and turn off whatever new messages you find annoying :D